src/Network/Network/Models/PSNetworkInterfaceIpConfiguration.cs

```diff
@@ -43,6 +43,9 @@ public class PSNetworkInterfaceIPConfiguration : PSIPConfiguration
         [JsonProperty(Order = 2)]
         public List<PSVirtualNetworkTap> VirtualNetworkTaps { get; set; }
 
+        [JsonProperty(Order = 2)]
+        public PSIpConfigurationConnectivityInformation PrivateLinkConnectionProperties { get; set; }
```

Comment:
Please add test that asserts this property. I'm not sure how automapper will handle it